apps/explorer/src/app/components/blocks/block-data.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { BlockMeta } from '../../routes/blocks/tendermint-blockchain-response';
+import { Link } from 'react-router-dom';
+import { SecondsAgo } from '../seconds-ago';
+import { Table, TableRow, TableCell } from '../table';
+
+interface BlockProps {
+  block: BlockMeta;
+  className?: string;
+}
+
+export const BlockData = ({ block, className }: BlockProps) => {
+  return (
+    <Table
+      aria-label={`Data for block ${block.header?.height}`}
+      className={className}
+    >
+      <TableRow dataTestId="block-row" modifier="background">
+        <TableCell
+          dataTestId="block-height"
+          className="pl-4 py-2"
+          aria-label="Block height"
+        >
+          <Link
+            to={`/blocks/${block.header?.height}`}
```

Comment:
chore (blocking): Instead of using `/blocks` hard codded can we use the `ROUTES.Blocks` from the router config so if we need to change routes it's only in a single place

Reply:
Other isntances of this in other places in the PR